requirements.txt

```diff
@@ -26,3 +26,7 @@ cmsis-pack-manager>=0.2.3,<0.3.0
 pywin32==224;platform_system=='Windows'
 wmi==1.4.9;platform_system=='Windows'
 psutil==5.6.2
+cryptography>=2.4.2
```

Comment:
shouldn't be more strict with versions ? By reviewing older dependencies here, you can notice we are more strict with minor/major updates - they break often, so we should not just allow any future version in

Reply:
License file should be also updated (what licenses are used)

Reply:
Which license file you mean?

We have license information in each file of ./sb-tools folder and also a file called LICENSE in .\sb-tools\prebuild, which contains text of "Permissive Binary License" that is applicable to our CyBootloader hex files.

Reply:
LICENSE.md file in the root has license of python modules listed

Reply:
Ohhh, okay should i made this update as part of merge request?

Reply:
Yes should be

Reply:
@0xc0170 Hi, i added license info for our python packages